src/dataobject.cpp

```diff
@@ -186,10 +186,10 @@ void dataobject::idata_row() {
 
 void dataobject::copy_parameters(int this_row, odeproblem* prob) {
   size_t n = par_from.size();
-  for(size_t i=0; i < n; ++i) {
+  for(size_t i = 0; i < n; ++i) {
     prob->param(par_to[i],Data(this_row,par_from[i]));
   }
-  prob->lsoda_init();
+  if(n > 0) prob->lsoda_init();
```

Comment:
Doesn't `any_copy` control this? This is redundant if `copy_parameters` is called only when `any_copy==true`.

Reply:
Yeah; thanks.